src/Cli/TerminusCommands.php

```diff
@@ -0,0 +1,158 @@
+<?php
+
+namespace UpdateTool\Cli;
+
+use Psr\Log\LoggerAwareInterface;
+use Psr\Log\LoggerAwareTrait;
+use Robo\Common\ConfigAwareTrait;
+use Robo\Contract\ConfigAwareInterface;
+use Hubph\HubphAPI;
+use Hubph\Git\WorkingCopy;
+use Hubph\Git\Remote;
+
+/**
+ * Commands used to interact with Terminus releases.
+ */
+class TerminusCommands extends \Robo\Tasks implements ConfigAwareInterface, LoggerAwareInterface
```

Comment:
Maybe `UpdateTerminusDocsCommands` would be clearer.

Reply:
Fixed